src/service/performance-impl.js

```diff
@@ -117,8 +117,14 @@ export class Performance {
      */
     this.aggregateShiftScore_ = 0;
 
+    // TODO(ccordry): there is a lot of other unnecessary work happening in
+    // this class for inabox case, but will require a larger refactor.
+    const isInabox = getMode(this.win).runtime === 'inabox';
```

Comment:
@jridgewell I am fighting with the linter here. I see this example https://github.com/ampproject/amphtml/blob/d55dbd91d0af315d4e6037ce5e3c1bef15dbfd34/build-system/eslint-rules/get-mode-usage.js#L23

Another lint rule wants me to destructure this prop if its the same var name so I tried
```js 
const {runtime} = getMode(this.win);
``` 
and 
``` js 
const runtimeType = getMode(this.win).runtime; 
``` 
but linter doesn't seem to like either. Am I doing something wrong?

Reply:
Never mind this was my mistake, it was complaining about the test file not the actual impl.